ci/ray_ci/windows/build_ray.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+set -ex
+
+git config --global core.symlinks true
+git config --global core.autocrlf false
+mkdir -p /c/rayci
+git clone . /c/rayci
+cd /c/rayci
+
+powershell ci/pipeline/fix-windows-bazel.ps1
+
+{
+  echo "build --announce_rc";  
+  echo "build --config=ci";
+  echo "startup --output_user_root=c:/tmp";
```

Comment:
should we use another directory for user root, rather than tmp? like maybe `c:/ray-work` or `c:/bazel`?

Reply:
this is because windows have a max path limit of 256 characters,  and the default I think is c:\Program Files\...; ray-work or bazel should work yes